vector/src/main/java/im/vector/app/features/call/webrtc/WebRtcCallManager.kt

```diff
@@ -88,11 +104,25 @@ class WebRtcCallManager @Inject constructor(
     private var peerConnectionFactory: PeerConnectionFactory? = null
     private val executor = Executors.newSingleThreadExecutor()
     private val dispatcher = executor.asCoroutineDispatcher()
+    var supportedPSTNProtocol: String? = null
+        private set
+
+    val supportsPSTNProtocol: Boolean
+        get() = supportedPSTNProtocol != null
 
     private val rootEglBase by lazy { EglUtils.rootEglBase }
 
     private var isInBackground: Boolean = true
 
+    init {
+        GlobalScope.launch {
+            supportedPSTNProtocol = currentSession?.getSupportedPSTN(3)
+            if (supportedPSTNProtocol != null) {
+                pstnSupportListeners.forEach { it.onPSTNSupportUpdated() }
```

Comment:
would be safer to catch callbacks failures

Reply:
Yup, thanks